src/warc2zim/main.py

```diff
@@ -309,12 +488,39 @@ def warc2zim(args=None):
 
     parser.add_argument(
         "-u",
-        "--main-url",
+        "--url",
         help="""The main url that should be loaded in the viewer on init""",
-        default="https://example.com/",
     )
 
-    parser.add_argument("-o", "--overwrite", action="store_true")
+    parser.add_argument(
+        "-a",
+        "--include-all",
+        action="store_true",
+        help="If set, include all URLs in ZIM, not just those specified in --include-domains",
+    )
+
+    parser.add_argument(
+        "-i",
+        "--include-domains",
+        action="append",
+        help="List of domains that should be included. Not used if --include-all is set. Defaults to domain of the main url",
+    )
+
+    parser.add_argument("-f", "--favicon", help="Favicon for Main Page")
```

Comment:
Not clear about what's expected here.

Reply:
Update help text and will now attempt to download the URL if not present in warc.